portfolio/src/main/webapp/script.js

```diff
@@ -53,27 +52,60 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-// Creates a map and adds it to the page with markers on compost-dropoff locations.
+// Global variables that apply across functions
```

Comment:
This comment, as is, is redundant. Perhaps describing why they're useful instead? Or feel free to leave them out.

Reply:
I decided to just delete the comment.